sql/src/main/java/org/apache/druid/sql/calcite/schema/SegmentMetadataCache.java

```diff
@@ -783,10 +789,13 @@ private Set<SegmentId> refreshSegmentsForDataSource(final String dataSource, fin
       yielder.close();
     }
 
+    long refreshDurationMillis = (System.nanoTime() - startNanos) / 1_000_000;
```

Comment:
Nit: for readability
```suggestion
    long refreshDurationMillis = TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - startNanos);
```